index.html

```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
             </li>
         </ul>
     </section>
+    <section class="works">
+        <h1 class="worksh">My Recent Works</h1>
+        <div class="divline"></div>
+        <section class="stories">
+            <img src="Art/stories-section.png" alt="Multi post stories">
+            <h2 class="multiposth">Multi-Post Stories</h2>
+            <p class="multipostp">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="languages">
+                <li class="cssbutt">
+                    <p class="icontext">css</p>
+                    <a></a>
+                </li>
+                <li class="htmlbutt">
+                    <p class="icontext">html</p>
+                    <a></a>
+                </li>
+                <li class="bootstrapbutt">
+                    <p class="icontext">bootstrap</p>
+                    <a></a>
+                </li>
+                <li class="rubybutt">
+                    <p class="icontext">Ruby</p>
+                    <a></a>
+                </li>
+            </ul>
+            <a class="seeproject">
+                <h3 class="seeprojecth">See Project</h3>
+            </a>
+        </section>
+        <section class="projectssection">
+            <div class="project1">
+                <h2 class="project1h">Professional Art Printing Data</h2>
+                <p class="project1p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="project1tags">
+                    <li class="htmlbuttp">
+                        <p class="icontextp">html</p>
+                        <a></a>
+                    </li>
+                    <li class="bootstrapbuttp">
+                        <p class="icontextp">bootstrap</p>
+                        <a></a>
+                    </li>
+                    <li class="rubybuttp">
+                        <p class="icontextp">Ruby</p>
+                        <a></a>
+                    </li>
+                </ul>
+                <ul>
+                    <li class="seeproject1">
+                        <h3 class="seeprojecttext">See Project</h3>
+                        <a></a>
+                    </li>
+                </ul>
```

Comment:
- [ ] The see project should be a button or anchor tag that users can click on 
- [ ] Please check Figma design and add button styling: Active, Hover and Pressed, check screenshot below ⬇️ 
![uuuuuuu](https://user-images.githubusercontent.com/69638013/159135294-e7e9b3f2-ebe2-4539-9b2e-a6549810f096.png)